docs/docs/plugin-authoring.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Plugin Authoring
+---
+
+One of the best ways to add functionality to Gatsby is through our plugin system. Gatsby is designed to be extensible, which means plugins are able to modify and extend just about everything Gatsby does.
+
+Of the many possibilities, plugins can:
+
+- add external data or content (e.g. your CMS, static files, a REST API) to your Gatsby GraphQL data
+- transform data from other formats (e.g. YAML, CSV) to JSON objects
+- add third-party services (e.g. Google Analytics, Instagram) to your site
+- anything you can dream up!
+
+## Core Concepts
+
+- Each Gatsby plugin can be installed as an npm package or as a [local plugin](#local-plugins)
```

Comment:
How about "Each Gatsby plugin an be created as an NPM package or as a..."

It seems odd to say you're "installing" a local plugin which you'd rarely do.